databases/package.json

```
@@ -0,0 +1,82 @@
+{
+  "name": "vscode-azuredatabases",
+  "author": "Microsoft Corporation",
+  "version": "0.0.1",
+  "description": "Common tools for developing Azure Databases support in Azure App Service extension for VS Code",
```

Since this package won't always be used in just App Service, I'd remove that part:

```suggestion
  "description": "Common tools for developing support for Azure Databases in VS Code",
```